x-pack/plugin/core/src/main/resources/monitoring-es.json

```
@@ -84,6 +84,19 @@
                           "type": "long"
                         }
                       }
+                    },
+                    "bulk": {
+                      "properties": {
+                        "total": {
```

I believe this should be:

```suggestion
                        "total_operations": {
```